code/modules/admin/verbs/debug.dm

```diff
@@ -357,76 +357,102 @@
 	for(var/areatype in areas_without_camera)
 		to_world("* [areatype]")
 
-/client/proc/cmd_admin_dress()
+/client/proc/cmd_admin_grab_observers()
 	set category = "Fun"
-	set name = "Select equipment"
+	set name = "Grab Observers"
+	set desc = "Grabs players that are observing and spawns them as basic humans beneath your feet."
+
+	var/list/chosen_observers = list()
+	var/next_observer = "NotGeeves"
+	while(next_observer != "== Finished ==")
+		var/list/valid_choices = player_list
+		for(var/choice in valid_choices)
+			if(choice in chosen_observers)
+				valid_choices -= choice
+			if(!istype(choice, /mob/abstract/observer))
+				valid_choices -= choice
+		valid_choices += "== Finished =="
+		next_observer = input("Choose an observer you want to add to the list.", "Choose Observer") as null|anything in valid_choices
+		if(!next_observer || isnull(next_observer))
+			next_observer = "== Finished =="
+		else
+			chosen_observers += next_observer
+	chosen_observers -= "== Finished =="
+
+	for(var/spawn_observer in chosen_observers)
+		var/mob/living/carbon/human/H = new /mob/living/carbon/human(get_turf(usr))
+		do_dressing(H)
```

Comment:
You sure you want this to be blocking? There's an input() in do_dressing()

Reply:
yup, in testing, it wasn't as bad as i thought it'd be